lib/context_functions_generator.ex

```diff
@@ -13,13 +13,19 @@ defmodule ContextFunctionsGenerator do
   def generate_function(:get, name, _pluralized_name, module, opts) do
     quote do
       def unquote(:"get_#{name}")(id, repo_opts \\ []) do
+        assocs = repo_opts[:assocs] || []
```

Comment:
what do you think about renaming `repo_opts` to just `opts`, since it now has more than just options for Repo